docker-compose.example.yml

```diff
@@ -1,3 +1,5 @@
+version: '3.9'
```

Comment:
`version` is obsolete - docker compose doesn't use it it's only informative so not needed in here: https://docs.docker.com/compose/compose-file/04-version-and-name/

Reply:
Very new change, maybe keep as informative, because use of healthcheck was re-introduced on 3.9, what you think? Remove or keep?

Reply:
I would ditch it altogether